src/scirpy/tl/_clonotypes.py

```diff
@@ -528,20 +533,40 @@ def clonotype_network(
     # explicitly annotate node ids to keep them after subsetting
     graph.vs["node_id"] = np.arange(0, len(graph.vs))
 
+    cell_indices = clonotype_res["cell_indices"]
+
     # store size in graph to be accessed by layout algorithms
-    clonotype_size = np.array([idx.size for idx in clonotype_res["cell_indices"].values()])
+    clonotype_size = np.array([len(idx) for idx in cell_indices.values()])
     graph.vs["size"] = clonotype_size
+
+    # create clonotype_mask for filtering according to mask_obs
+    if mask_obs is not None:
+        cell_mask = adata.obs[mask_obs]
+        cell_indices_reversed = {v: k for k, values in cell_indices.items() for v in values}
+        clonotype_mask = np.array([False] * len(cell_indices.items()))
```

Comment:
Not critical here, but this is more efficient ;)

```suggestion
        clonotype_mask = np.zeros((len(cell_indices), ), dtype=bool)
```